### DISCUSSION TITLE
Experiment label on the wrong `ax` object

### DISCUSSION THREAD

**Discussion by ico1036** [scikit-hep/mplhep#335] [Q&A]
Hi, I'm trying to draw a ratio plot with experiment label.
But the location of label is fixed on the ratio plot not main plot
How do I label it correctly?
I attach the plot and script.

Thank you in advance.

![image](https://user-images.githubusercontent.com/13959858/132791711-1963e51e-bfe7-4fcb-9f9d-70f7ec847099.png)

```python
import matplotlib.pyplot as plt
import mplhep as hep

plt.style.use(hep.style.CMS)


plt.rcParams.update({
    'font.size': 15,
    'axes.titlesize': 18,
    'axes.labelsize': 18,
    'xtick.labelsize': 14,
    'ytick.labelsize': 14
})


fig, (ax, rax) = plt.subplots(
    nrows=2,
    ncols=1,
    figsize=(10,10),
    gridspec_kw={"height_ratios": (3, 1)},
    sharex=True
)
hep.cms.label(data=True,lumi=59.97,year=2018,loc=0)
fig.subplots_adjust(hspace=.07)
```

**Comment by andrzejnovak** [comment]
Hi @ico1036, this should fix this for you 
```
hep.cms.label(data=True,lumi=59.97,year=2018,loc=0, ax=rax)
```

Otherwise `label` always picks up the "current" `ax` object, similarly to when you call `plt.stuff`.
